app/lti.go

```diff
@@ -4,15 +4,22 @@
 package app
 
 import (
-	"net/http"
 	"encoding/json"
 	"fmt"
+	"net/http"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 	"github.com/mattermost/mattermost-server/v5/shared/mlog"
 )
 
-func (a* App) GetLTISettings() (*model.LTISettings, error) {
+//
+//	GetLTISettings() reads the LTI Config from Plugin Config
+//	and unmarshal the JSON to LTISettings struct.
+//
```

Comment:
Explain what happens when
- LTI plugin is not installed
- LTI plugin is enabled/disabled

Reply:
what about this?